src/dns/implementation.ts

```diff
@@ -0,0 +1,8 @@
+import { DNS_IMPLEMENTATION } from "./implementation/dns.js"
```

Comment:
I think we should call this implementation "dns-over-http" or just plainly "http".

Reply:
Good call! I changed it over to "http".